src/core/lib/surface/call.cc

```diff
@@ -2349,16 +2351,17 @@ grpc_error_handle MakePromiseBasedCall(grpc_call_create_args* args,
                                        grpc_call** out_call) {
   Channel* channel = args->channel.get();
 
-  auto* arena = channel->CreateArena();
+  auto arena = channel->call_arena_allocator()->MakeArena();
   PromiseBasedCall* call = arena->New<T>(arena, args);
```

Comment:
`std::move()`?

Reply:
done

Reply:
Oh I made it, it had a compile error, and I backed it out.

This code is going to be deleted shortly anyway (I have the PR typed), so any micro-optimizations are really not worth pursuing.